### PR TITLE
feat: add "skip projects" button

### PR DESCRIPTION
**Pull request by rbondoc96**
Issue #59 

- [x] Created a SkipButton component
- [x] Added a SkipButton with text "Skip Projects" to top left of "Projects" section
- [x] Added ".section-wrapper" to wrap around a <section> and its associated SkipButton

### CODE REVIEW

File: src/components/SkipButton.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+            <div className="skip-proj-btn-wrapper">
+                <a href={`#${skipTo}`} className="skip-proj-btn secondary-btn">
+                Skip Projects
+                </a>     
+            </div>          
+        </>
+    )
+}
+
+export default SkipButton;
```
Comment (Ngoakor12) [inline src/components/SkipButton.js]: Can you please rename this component/file "SkipSectionButton", that way it's more specific and can possibly be reused for any sections that need a skip button(e.g. navigation)

File: src/components/SkipButton.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+function SkipButton({ skipTo }) {
+    return(
+        <>
+            <div className="skip-proj-btn-wrapper">
+                <a href={`#${skipTo}`} className="skip-proj-btn secondary-btn">
+                Skip Projects
+                </a>     
+            </div>          
+        </>
```
Comment (Ngoakor12) [inline src/components/SkipButton.js:9]: The fragments seem redundant too, remove them since they're not doing anything.

**Review by Ngoakor12 — CHANGES_REQUESTED** [review]
Hey, thanks again for working on this.

Everything looks and works as I expected. Great work bro. I don't mind repetitive styles and new media queries, my CSS file is a mess and needs refactoring.

**I do have a few requests below:**

File: src/components/SkipButton.js (hunk 2 above)
Comment (rbondoc96) [reply src/components/SkipButton.js]: Sure thing. I've done the following:

- [x]: Renamed SkipButton to SkipSectionButton
- [x]: Renamed classes to skip-section-btn
- [x]: Removed fragments from SkipSectionButton

**Comment by Ngoakor12** [comment]
@rbondoc96 Are you still working on this PR or should I or someone else take over?

**Comment by rbondoc96** [comment]
> @rbondoc96 Are you still working on this PR or should I or someone else take over?

@Ngoakor12 Hey, sorry about that. I thought I had committed it. I pushed changes over with the `pointer-events` changes

**Review by Ngoakor12 — APPROVED** [review]
It now works as expected, thanks again for your contribution.

**Pull request merged**